server/db/schema/events.sql

```diff
@@ -0,0 +1,24 @@
+-- DROP TABLE IF EXISTS Events;
+-- DROP TABLE IF EXISTS Classes;
+
+DROP TYPE IF EXISTS level;
+CREATE TYPE level AS ENUM ('beginner', 'intermediate', 'advanced');
+
+-- CREATE TABLE Classes (
+--     id SERIAL PRIMARY KEY
+-- );
+
+CREATE TABLE Events (
+    id SERIAL PRIMARY KEY,
+    location VARCHAR(256) NOT NULL,
+    title VARCHAR(256) NOT NULL,
+    description TEXT,
+    level LEVEL NOT NULL,
+    date DATE NOT NULL,
+    start_time TIME NOT NULL,
+    end_time TIME NOT NULL,
+    call_time TIME NOT NULL,
+    class_id INTEGER NOT NULL,
+    FOREIGN KEY (class_id) REFERENCES Classes(id),
```

Comment:
We also want to include cascade on delete, in the event that the row that we are referencing gets deleted, then we can delete this row as well.

This is more of a NIT but you should also just list out all the columns first before adding in any constraints.